internal/database/autocontact_test.go

```diff
@@ -0,0 +1,152 @@
+package database
+
+import (
+	"log/slog"
+	"os"
+	"testing"
+	"time"
+
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+)
+
+func TestSSHRestart(t *testing.T) {
+	// Setup
+	logger := slog.Default()
+	sshConfig := SSHConfig{
+		User:    "testuser",
+		Keypath: "/path/to/your/test/private/key",
+	}
+
+	err := sshRestart("invalid.remote.address", logger, sshConfig)
+	if err == nil {
```

Comment:
This needs to be much more specific, as currently all the ssh tests just want a non-nil error.

Take a look at `errors.Is` and `errors.As` to try to find the error you expect here.